configsets/site-search/conf/solrconfig.xml

```diff
@@ -930,41 +930,6 @@
     <arr name="components">
       <str>suggest</str>
     </arr>
-  </requestHandler> -->
```

Comment:
I assume this was residual, and not needed.

Reply:
Yep, it's a default config thing that we aren't using and is just eating (a small amount) of memory for no reason.